autogen.sh

```diff
@@ -1,6 +1,42 @@
 #!/bin/sh
 
+usage()
+{
+	echo "Usage: autogen.sh <options>"
+	echo
+	echo "  -h|--help         Show this help message"
+	echo "  --with-ucg        Fetch UCG submodule"
+	echo
+}
+
+with_ucg="no"
+
+for key in "$@"
+do
+	case $key in
+	-h|--help)
+		usage
+		exit 0
+		;;
+	--with-ucg)
+		with_ucg="yes"
+		;;
+	*)
+		usage
+		exit -2
+		;;
+	esac
+done
+
 rm -rf autom4te.cache
-mkdir -p config/m4 config/aux 
+mkdir -p config/m4 config/aux
+
+if [ "X$with_ucg" = "Xyes" ]
+then
+	git submodule update --init --recursive
+else
+	mkdir -p src/ucg
```

Comment:
why it is needed?

Reply:
after the last changes in Makefile.am, it's not needed anymore

Reply:
@alex--m can you remove this one please?